argoverse/utils/calibration.py

```diff
@@ -235,6 +236,34 @@ def load_calib(calib_filepath: Union[str, Path]) -> Dict[Any, Calibration]:
     return calib_list
 
 
+def load_stereo_calib(calib_filepath: Union[str, Path]) -> Dict[Any, Calibration]:
```

Comment:
Could we change `calib_filepath: Union[str, Path]` -> `calib_filepath: _PathLike` for consistency? See `map_api.py` for an example. Eventually, we should move the typing into one place rather than copying type aliases everywhere (as seen in `map_api.py`).

Reply:
I think we can do that in another PR as I've seen that typing style in other functions too.